index.rst

```diff
@@ -5,17 +5,19 @@ Welcome to the official documentation of the UBports project!
 
 UBports develops the mobile phone operating system Ubuntu Touch. Ubuntu Touch is a mobile operating system focused on ease of use, privacy, and convergence.
 
-On this website you find instructions how to install Ubuntu Touch on your mobile phone, user guides and detailed documentation on all system components. If this is your first time here, please consider reading our :doc:`introduction <about/introduction>`.
+On this website you find instructions how to install Ubuntu Touch on your mobile phone, user guides and detailed documentation on all system components. If this is your first time here, please consider reading our :doc:`introduction <general/about_ubuntu_touch>`.
```

Comment:
Why? It makes more sense to show people the introduction to the docs. You could additionally add a link to ubuntu-touch.io (info about the product) and ubports.com (info about the project), but what's important here is the introduction to the documentation page.

Reply:
I agree and I've created a pad and am writing a fix now.  I think I missed it because its the index ;P  Sorry everyone.  I'll have something new shortly.